LICENSE.AGPL

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
@nikkolasg Not sure which exact meeting you are referring to, but yes I can remember @bford proposing an optional (ASCII) field for additional (experimental) protocols. But as far as I understood it, this was only about the *data* that will be send around for that protocol and not about potential validation functionality on this data. Were you aware that this means that ideally one can add protocols (messages and methods that validate/act on this data) without modifying the underlying (golang) code? But I agree, let's discuss the details in the meetings 👍